support/two-types-of-work.md

```diff
@@ -0,0 +1,22 @@
+# Two types of work
+
+There are two types of work in IT Operations.
+
+## Changes (Planned work)
+
+?> Scheduled work or 'Operational changes' are fixed and generally planned improvements that should be tested, communicated and actioned with the most care you can think of.
```

Comment:
I'm not quite clear on what these are; are they the result of project work or is this more like software package upgrades?

Reply:
Could be project work but more referring to upgrades, migrations, network changes and such - planned changes rather than users submitting bugs/issues